wrappers/golang/curves/bw6761/scalar_field.go

```diff
@@ -0,0 +1,119 @@
+package bw6761
```

Comment:
This is exactly the same field as bls12_377 scalar field but if it's easier to redefine than to re-use, I don't see it as a huge issue.

Reply:
I'll add it as a task but it was easier due to templates just to redefine it